docusaurus.config.js

```diff
@@ -41,6 +41,12 @@ module.exports = {
 					to: '/',
 					label: 'Docs',
 					position: 'left',
+					activeBaseRegex: '^((?!/user-guide/rules).)*$',
```

Comment:
This is necessary since without it, any path that starts with `/user-guide/rules` matches **both**:

- the docs path
- the rules path

which leads to a double-highlight, i.e. both links are highlighted as active. I set a "inverse match" (really, negative lookbehind I think?) to avoid matching the "Docs" active link when the user is on the rules page.

Let me know if this is confusing and/or if we'd prefer an alternative!

Reply:
Good consideration! 👍🏼 

In addition, I think we may improve the regex like this, but any thoughts?
(`*` may worse its performance)

```suggestion
					activeBaseRegex: '^(?!/user-guide/rules)',
```

Furthermore, how about excluding the `/demo` page, too?

```suggestion
					activeBaseRegex: '^(?!/user-guide/rules|/demo)',
```

Reply:
Great suggestion, thanks!